Gems/Atom/Tools/MaterialEditor/Code/Source/Window/MaterialEditorWindow.cpp

```diff
@@ -304,7 +304,7 @@ namespace MaterialEditor
             }
         }, QKeySequence::New);
 
-        m_actionOpen = m_menuFile->addAction("&Open...", [this]() {
+        m_actionOpen = m_menuFile->addAction("&Open...", []() {
```

Comment:
I guess we are OK with the messed up indentation?

Reply:
yeah, didnt want to get into that...